error/src/lib.rs

```diff
@@ -436,7 +435,7 @@ impl CaliptraError {
         CaliptraError::new_const(0x000E0039);
     pub const RUNTIME_PRIV_KEY_KV_HDL_HANDOFF_FAILED: CaliptraError =
         CaliptraError::new_const(0x000E003A);
-    pub const RUNTIME_HASH_CHAIN_HANDOFF_FAILED: CaliptraError =
+    pub const FIRMWARE_HASH_CHAIN_HANDOFF_FAILED: CaliptraError =
```

Comment:
I think this should still have the `RUNTIME` prefix since it's a runtime error.

Reply:
Fixed.